### PR TITLE
Forward compatibility with Stream v0.7, v0.6, v0.5 and upcoming v1.0 (while keeping BC)

### PR DESCRIPTION
**Pull request by thklein**

### CODE REVIEW

**Comment by clue** [comment]
Nice! Does it make sense to cover upcoming v0.7 and v1.0 as well as part of this PR or should tackle this separately? :+1:

**Review by clue — APPROVED** [review]

**Comment by thklein** [comment]
Yes, it makes sense for the upcoming versions as well. Will be covered in this PR

**Comment by thklein** [comment]
Updated...

File: README.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ The recommended way to install this library is [through Composer](https://getcom
 This will install the latest supported version:
 
 ```bash
-$ composer require clue/ndjson-react:^0.1
+$ composer require clue/ndjson-react:^0.7
```
Comment (clue) [inline README.md]: Unrelated change?

Comment (thklein) [reply README.md]: Already fixed!

**Comment by clue** [comment]
Nice, thanks! :shipit:
